### DISCUSSION TITLE
Practica 1 - Lo que entendi

### DISCUSSION THREAD

**Discussion by ElPoder1** [Launch-X-Latam/MisionFrontEnd#129] [Q&A] [answered]
Recién revise el contenido de practica y esto es _lo que considero se debe hacer en cada paso_, lo comento por si a alguien le sirve, si me salto algo o entendí mal con gusto podemos colaborar. salu2.

**1. Toma de requerimientos**
Aquí nos presentan un formato el cual es una formalidad "cliente/vendedor", en si es una base que nos sirve para desarrollar cualquier proyecto que nos presenten y formalizarnos, debemos completarlo con los datos para Autobot

**1.1 Buyer Persona**
Aquí debemos definir a nuestro cliente ideal, la platilla mostrada es sobre como debemos describir a nuestro cliente, supongo que es el aboga pues en el punto 2 habla del publico objetivo

**2. Publico Objetivo**
Debemos definir el publico objetivo y para ello nos podemos apoyar en la pagina que nos muestran

**3. Wireframe UX**
Debemos crear el wireframe que nos servirá de base para la interface UI

**4. Interface UI**
Debemos crear la interface de modo que cumpla con lo requerido en la descripción del Abogabot

**_Actualización_**
Pienso subir un repositorio en mi Github donde subo cada práctica pero estoy investigando como programar los .md ¿Así se podrá entregar?

**Comment by doguedogue** [comment, marked as answer]
Exactamente, tienes toda la razón, gracias por compartir 🚀🚀🚀

**Comment by Evelo00** [comment]
No tenia muy claro lo de la toma de requerimientos, gracias!

**Comment by EliEsquivel2802** [comment]
Muchas gracias!! estaba perdida la verdad es que no se casi nada de programación todo es nuevo y estaba muy confundida de cómo hacer las practicas

**Comment by Leslie1520** [comment]
Gracias por compartir una explicación :)
